app/controllers/v1/tasks_controller.rb

```diff
@@ -0,0 +1,114 @@
+module V1
+  class TasksController < ApplicationController
+    swagger_controller :tasks, 'tasks'
+
+    swagger_api :index do
+      summary 'List all tasks'
+      notes 'This lists all the active tasks'
+      param :path, :project_id, :string, :required, 'Project Id'
+      param :query, :page, :integer, :optional, 'page number of results, default 1'
+      param :query, :page_size, :integer, :optional, 'number of results per page, default 25'
+    end
+    def index
+      tasks, errors = ListTasks.new(index_params).call
+
+      if errors.any?
+        render json: { errors: errors }, status: 400
+      else
+        render json: tasks
+      end
+    end
+
+    swagger_api :show do
+      summary 'Fetch a single task'
+      param :path, :project_id, :string, :required, 'Project Id'
+      param :path, :id, :string, :required, 'Task Id'
+    end
+    def show
+      task = project.tasks.find_by id: params[:id]
```

Comment:
I like how you made `project` a memoized method.
You have this line for finding the task through the project 3 times. It might also be good to use a method as well. That being said, I can see why you may not have since you have a different `task` in the create method.